tuf/settings.py

```diff
@@ -74,12 +74,6 @@
 # download Targets metadata.
 DEFAULT_TARGETS_REQUIRED_LENGTH = 5000000 #bytes
 
-# Set a timeout value in seconds (float) for non-blocking socket operations.
-SOCKET_TIMEOUT = 4 #seconds
-
-# The maximum chunk of data, in bytes, we would download in every round.
-CHUNK_SIZE = 400000 #bytes
-
```

Comment:
These are settings that unfortunately maybe used by someone so I don't think we should just silently delete them without deprecation.

I wonder if the new implementation should still use these as the default values for its properties... and possibly mark these as deprecated, explaining that these values are now fetcher specific and that the default RequestsFetcher exposes the same properties.

Reply:
I totally forgot about `settings` being part of the API. I dropped the commit and just used `tuf.settings` in `RequestsFetcher`. 

I suggest we revise `tuf.settings` in general and then apply the decisions here.